fastapi_websocket_rpc/websocket_rpc_client.py

```diff
@@ -137,25 +280,6 @@ async def __connect__(self):
                     await self.channel.close()
                 self.cancel_tasks()
                 raise
-        except ConnectionRefusedError:
-            logger.info("RPC connection was refused by server")
-            raise
-        except ConnectionClosedError:
-            logger.info("RPC connection lost")
-            raise
-        except ConnectionClosedOK:
-            logger.info("RPC connection closed")
-            raise
-        except InvalidStatusCode as err:
-            logger.info(
-                f"RPC Websocket failed - with invalid status code {err.status_code}")
-            raise
-        except WebSocketException as err:
-            logger.info(f"RPC Websocket failed - with {err}")
-            raise
-        except OSError as err:
-            logger.info("RPC Connection failed - %s", err)
-            raise
         except Exception as err:
             logger.exception("RPC Error")
```

Comment:
When `connect` raises an error - the exception (with stack trace) is logged twice

Reply:
Nice catch.
@antoinetran - one last fix and I think we can merge! :) 

P.s.
Can you DM me / email me (or@permit.io) your shipping address - will send you some swag- if you want :)

Reply:
This is fixed: https://github.com/antoinetran/fastapi_websocket_rpc/commit/3e985f6f0e43746daff64860ee3f2473f3fd382a